kernel/kernbench.py

```diff
@@ -120,8 +120,10 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % package)
         self.kernel_version = platform.uname()[2]
         self.iterations = self.params.get('runs', default=1)
-        self.threads = self.params.get(
-            'cpus', default=2 * cpu.online_cpus_count())
+        self.online_cpu = 2 * cpu.online_cpus_count()
+        self.threads = self.params.get('cpus', default=None)
+        if self.threads is None:
+            self.threads = self.online_cpu
```

Comment:
Sorry for not noting it first time around, I think the use of `self.online_cpu` variable can be avoided by assigning the value directly here.

Reply:
I removed the self.online_cpu variable and assigned the value directly to self.threads